postgres-appliance/scripts/configure_spilo.py

```diff
@@ -423,7 +422,7 @@ def set_extended_wale_placeholders(placeholders, prefix):
 
 def set_walg_placeholders(placeholders, prefix=''):
     walg_supported = any(placeholders.get(prefix + n) for n in AUTO_ENABLE_WALG_RESTORE +
-                         ('WAL_GS_BUCKET', 'WALE_GS_PREFIX', 'WALG_GS_PREFIX'))
+                         ('WAL_GS_BUCKET', 'WALE_GS_PREFIX', 'WALG_GS_PREFIX', 'WALG_AZ_PREFIX'))
```

Comment:
This change is not required, `AUTO_ENABLE_WALG_RESTORE` already contains `WALG_AZ_PREFIX`.